docs/Reference/API-Methods.md

```diff
@@ -2049,6 +2049,52 @@ is irrelevant to the estimation process (unlike transactions, in which gas limit
         }
         ```
 
+### `eth_feeHistory`
```

Comment:
What's the use case for this API?

Reply:
It gives the base fee history and gas used/gas limit ratio history over a range, so it could let you track trends and fluctuations over time.  Do I need to elaborate the description any?

Reply:
I'd work your description above into the intro.